src/styles/navigation.scss

```diff
@@ -0,0 +1,55 @@
+.navigation {
+  margin-right: 50px;
+  font-family: "Roboto", sans-serif;
+
+  &__items-list {
+    display: flex;
+    align-items: center;
+    height: 60px;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+
+    &__item {
```

Comment:
An example of grandchildren element, don't use element inside element in BEM, "&__item" can bem an element of .navigation!

[Grandchildren usecase example](https://mate-academy.github.io/style-guides/htmlcss.html#bem-element-nesting)